lib/cluster.js

```diff
@@ -104,11 +104,11 @@ function RoundRobinHandle(key, address, port, addressType, backlog, fd) {
   this.server = net.createServer(assert.fail);
 
   if (fd >= 0)
-    this.server.listen({ fd: fd });
+    this.server.listen({ fd: fd }, backlog);
   else if (port >= 0)
-    this.server.listen(port, address);
+    this.server.listen(port, address, backlog);
   else
-    this.server.listen(address);  // UNIX socket path.
+    this.server.listen(address, backlog);  // UNIX socket path.
```

Comment:
According to server.listen's implementation, if second or third argument
  is a Number, treat it as backlog size.

  But server.listen(options[, callback]) already have a backlog in options
  and has higher priority, it is better to avoid document it has another
  standalone backlog parameter to avoid more confuse.